pkg/executor/fscache/functionServiceCache.go

```diff
@@ -173,6 +205,23 @@ func (fsc *FunctionServiceCache) GetByFunctionUID(uid types.UID) (*FuncSvc, erro
 	return &fsvcCopy, nil
 }
 
+func (fsc *FunctionServiceCache) AddFunc(fsvc FuncSvc) {
+	fsc.connFunctionCache.Set(crd.CacheKey(fsvc.Function), fsvc.Address, &fsvc)
+	now := time.Now()
+	fsvc.Ctime = now
+	fsvc.Atime = now
+
+	fsc.setFuncAlive(fsvc.Function.Name, string(fsvc.Function.UID), true)
+}
+
+func (fsc *FunctionServiceCache) GetActiveInstances(m *metav1.ObjectMeta) int {
+	return fsc.connFunctionCache.GetTotalActive(crd.CacheKey(m))
+}
+
+func (fsc *FunctionServiceCache) SetInActive(fn *fv1.Function, svcHost string) {
+	fsc.connFunctionCache.UnSet(crd.CacheKey(&fn.ObjectMeta), svcHost)
+}
```

Comment:
Let's use `Active` --> `Available` in all places?

Reply:
Done